Sources/Backtrace/Backtrace.swift

```diff
@@ -13,7 +13,8 @@ public enum Backtrace {
             let ptr = backtrace_symbols(stackSymbols.baseAddress!, howMany)
             let realAddresses = Array(UnsafeBufferPointer(start: ptr, count: Int(howMany))).compactMap { $0 }
             realAddresses.forEach {
-                print(String(cString: $0))
+                fputs($0, stderr)
+                fputc(0x0A, stderr)
```

Comment:
Isn't `$0` already null-terminated? Shouldn't this be unnecessary?

Reply:
Sorry, I'm being silly. `0x0 != 0xA` :gri

Reply:
It might be better to combine this into one call, that way we eliminate the possibility of another thread writing to stderr between the string and the newline.

```swift
fprintf(stderr, "%s\n", $0)
```

Reply:
A good point, although I'm not sure `fprintf` is available to Swift due to its varargs :( This is why I went with this manual approach at first. But I'll look into it later.

Reply:
Thanks! If it's not, you could add a helper to `CBacktrace/shim.h`?:

```c
static inline void cbacktrace_fprintf_stderr_helper(const char *str) {
    fprintf(stderr, "%s\n", str);
}
```

Reply:
What do you think of using `backtrace_symbols_fd`, by the way? That could be another approach to write everything out.

Reply:
I'm happy to use `backtrace_symbols_fd()` instead, in the current design.

If/when we redesign this package to be more featureful we will want to be able to cache backtraces and pass them around, which `backtrace_symbols_fd()` won't be suitable for. But for now it sounds fine :)